.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5502
+}
```

Comment:
- [ ] Kindly remove all unnecessary from your project, as it takes up extra space. You can delete or add it to the `.gitignore` file.